### DISCUSSION TITLE
Replacing a dot with a comma

### DISCUSSION THREAD

**Discussion by GuilhermeHirsch** [Budibase/budibase#2173] [Help] [answered]
Hi! I'm trying to set currency values. So I'm trying to replace a dot with a comma like this, but something isn't right:

![Sem título](https://user-images.githubusercontent.com/33108596/127490625-9837dcb5-211b-49d8-ba0a-cf5f075fcff9.jpg)

Any help on that? Thank you so much for everything, guys!

**Comment by shogunpurple** [comment, marked as answer]
@GuilhermeHirsch thank you for the detailed report!

I can't seem to recreate this one - I have a table called `Money`, that has a column called `Money ValueÇ`. I called it this because I wanted to see if the space in the column name, or even the portuguese cedilla was a problem because of the encoding. I set the value of this column to `1.000.000` in a row, then tried to bind it in the design section. Here's what my design section looks like:

<img width="1644" alt="Screenshot 2021-07-29 at 14 43 03" src="https://user-images.githubusercontent.com/11256663/127502756-8ae17864-83e9-4d78-aa97-7d93952015ce.png">

What does the structure of your app look like? Is this an older app, or a new one?

**Reply by GuilhermeHirsch** [reply]
Hi, @shogunpurple! Thanks for the quick support. My structure is this:

https://www.loom.com/share/a7b55b460d8f45f2a41255446990cecf

This app has been updated

**Reply by shogunpurple** [reply]
Ah - I see it. Thanks again for the detail. The issue is that your column value is a number, and I am using a string. It looks like you will need to parse your number to a string first then do the replace. Something like this:

`{{ replace ( toPrecision New Repeater.Money.Value 10) "." "," }}`

Let me know if that helps!

**Reply by GuilhermeHirsch** [reply]
It worked, however, that way, I need to define the number of digits, which can sometimes vary.

![Sem título2](https://user-images.githubusercontent.com/33108596/127512028-355bfc99-3bcc-4403-80c7-067f0c997f44.jpg)

![Sem título1](https://user-images.githubusercontent.com/33108596/127512044-fd8f12a1-5d69-4337-afb2-f7291dd9b694.jpg)

![Sem título](https://user-images.githubusercontent.com/33108596/127512061-2abaac3e-6764-4260-94b5-0f3f4db798ab.jpg)

**Reply by PGLange** [reply]
Hey @GuilhermeHirsch 
try {{toFixed (replace(...))  3}}, where 3 is the count of digits after the dot. 
Hope this helps

**Reply by shogunpurple** [reply]
I've found another workaround that should circumvent this altogether:

`{{ replace ( toPath New Repeater.Money.ValueNumber) "." "," }}`

**Reply by GuilhermeHirsch** [reply]
Worked perfectly, @shogunpurple and @PGLange! Thank you very much! I didn't find "toPath" among the helpers. I used both solutions: Valor Total: R$ {{ replace ( toFixed ( toPath New Repeater.Cobranças.VALOR TOTAL) 2 ) "." "," }}
